docs/en/deployment/quick-start.md

```diff
@@ -1,25 +1,36 @@
-In order to let you get started with Apollo Configuration Center faster, we have prepared a Quick Start here, which can deploy and start Apollo Configuration Center in your local environment in a few minutes.
+In order to let you get started with Apollo Configuration Center faster, we have prepared a Quick Start here, which can
+deploy and start Apollo Configuration Center in your local environment in a few minutes.
```

Comment:
why split it to two lines?

Reply:
@nobodyiam This is to ensure that the line length does not become too long. When rendered, these two lines will still appear as one line.

Reply:
In my view, this isn't essential, but if we choose to format the markdown files, we should format all of them rather than just one. Also, note that Apollo's [code style](https://github.com/apolloconfig/apollo/blob/master/apollo-buildtools/style/intellij-java-google-style.xml#L44) sets the line limit to 100, whereas it appears your IDE is utilizing 120.

Reply:
@nobodyiam I agree.